Makefile

```diff
@@ -206,8 +213,9 @@ CFLAGS=-DLOC_FILE_INDEX=LOC_$(subst .,_,$(subst -,_,$(notdir $<)))
 # Use this recursively defined variables to build the path to pick-up both
 # forms of #include .h files.
 # -----------------------------------------------------------------------------
-INCLUDE = -I ./$(UNIT_INCDIR)
+INCLUDE := -I ./$(UNIT_INCDIR)
 INCLUDE += -I ./$(dir $<)
+INCLUDE += -I ./$(INCDIR)
```

Comment:
The way these INCLUDE path will work is:

- If user is building an example program that requires generated files (i.e. uses LOC scheme), the include the generated `loc.h` from the example program's source-dirs. Previous rules have selected that the generated `loc.h` should be done in the dirs for the example program.
- If not, user is using LOC2 scheme. In that case, #include the checked-in `include/loc.h` file. 

This way, we select only either one of the files.

This is all just build-magic for the tests in this repo which exercise both techniques.

In user's build-env, these rules will need to be adopted / adapted slightly.